doc/source/developer_guide/dg_forcing.rst

```diff
@@ -0,0 +1,216 @@
+- Data is read on the model grid, no spatial interpolation exists.
+- Data is often time interpolated linearly between two input timestamps to the model time each model timestep.
+
+In general, the following variables need to be defined by the forcing module,
+
+From Atmosphere:
+
+- zlvl    = atmosphere level height (m)
+- uatm    = model grid i-direction wind velocity component (m/s)
+- vatm    = model grid j-direction wind velocity component (m/s)
+- strax   = model grid i-direction wind stress (N/m^2)
+- stray   = model grid j-direction wind stress (N/m^2)
+- potT    = air potential temperature  (K)
+- Tair    = air temperature  (K)
+- Qa      = specific humidity (kg/kg)
+- rhoa    = air density (kg/m^3)
+- flw     = incoming longwave radiation (W/m^2)
+- fsw     = incoming shortwave radiation (W/m^2)
+- swvdr   = sw down, visible, direct  (W/m^2)
+- swvdf   = sw down, visible, diffuse (W/m^2)
+- swidr   = sw down, near IR, direct  (W/m^2)
+- swidf   = sw down, near IR, diffuse (W/m^2)
+- frain   = rainfall rate (kg/m^2 s)
+- fsnow   = snowfall rate (kg/m^2 s)
+
+From Ocean:
+
+- uocn    = ocean current, x-direction (m/s)
+- vocn    = ocean current, y-direction (m/s)
+- ss_tltx = sea surface slope, x-direction (m/m)
+- ss_tlty = sea surface slope, y-direction (m/m)
+- hwater  = water depth for basal stress calc (landfast ice)
+- sss     = sea surface salinity (ppt)
+- sst     = sea surface temperature (C)
+- frzmlt  = freezing/melting potential (W/m^2)
+- frzmlt_init= frzmlt used in current time step (W/m^2)
+- Tf      = freezing temperature (C)
+- qdp     = deep ocean heat flux (W/m^2), negative upward
+- hmix    = mixed layer depth (m)
+- daice_da= data assimilation concentration increment rate (concentration s-1)(only used in hadgem drivers)
+
+All variables have reasonable but static defaults and these will be used in ``default`` mode.
+
+To advance the forcing, the subroutines **get_forcing_atmo** and
+**get_forcing_ocn** are called each timestep from the step
+loop.  That subroutine computes the forcing year (``fyear``), calls the appropriate
+forcing data method, and then calls **prepare_forcing** which converts the 
+input data fields to model forcing fields.
+
+.. _JRA55forcing:
+
+JRA55 Atmosphere Forcing
+-------------------------
+
+The current default atmosphere forcing for gx3, gx1, and tx1 standalone grids for
+Consortium testing is the JRA55 forcing
+dataset :cite:`Tsujino18`.  The Consortium has released 5 years of forcing data, 
+2005-2009 for gx3, gx1, and tx1 grids.  Each year is a separate file and 
+the dataset is on a gregorian time axis which includes leap days.
+
+.. _fig-jra55data:
+
+.. figure:: ./figures/jra55data.png
+   :align: center
+   :scale: 100%
+
+   Schematic of JRA55 CICE forcing file generation.
+
+The forcing is read and interpolated in subroutine **JRA55_data**.  In particular,
+air temperature (airtmp), east and north wind speed (wndewd and wndnwd), 
```

Comment:
could format the variable names in the code - not a big deal

Reply:
Updated and committed.